### PR TITLE
add ubeswap

### PR DESCRIPTION
**Pull request by sevenshi**
add ubeswap

### CODE REVIEW

File: subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+export class UbeswapCeloConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.CELO;
+  }
+  getProtocolName(): string {
+    return "Ubeswap";
+  }
+  getProtocolSlug(): string {
+    return "Ubeswap";
```
Comment (this-username-is-taken) [inline subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts:16]: can you change to all lowercase: `ubeswap`?

File: subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  getLPFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("2.5");
+  }
+  getProtocolFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("0");
+  }
+  getLPFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("3");
+  }
+  getFeeOnOff(): string {
```
Comment (steegecs) [inline subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts:49]: Where did you see that this fee starts on? I have just seen that 0.3% goes to the liquidity providers and eventually some will be distributed to token holders.

File: subgraphs/uniswap-forks/protocols/ubeswap/src/common/handlers/handleReward.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+): void {
+  // Return if pool does not exist
+  let pool = LiquidityPool.load(event.address.toHexString());
+  if (!pool) {
+    return;
+  }
+  let stakeToken = getOrCreatePair(pool.inputTokens[0]);
+
+  pool.stakedOutputTokenAmount = pool.stakedOutputTokenAmount!.plus(amount);
+  pool.inputTokenBalances[0] = pool.stakedOutputTokenAmount!;
```
Comment (steegecs) [inline subgraphs/uniswap-forks/protocols/ubeswap/src/common/handlers/handleReward.ts:33]: Why are you storing the staked token balance in the input tokens as well? To me, it make more sense to just leave the storage of the token pairs in the inputTokenBalances and then store the staked amount in the stakedOutputTokenAmount.

File: subgraphs/uniswap-forks/protocols/ubeswap/src/common/handlers/handleReward.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+): void {
+  //Return if pool does not exist
+  let pool = LiquidityPool.load(event.address.toHexString());
+  if (!pool) {
+    return;
+  }
+  let stakeToken = getOrCreatePair(pool.inputTokens[0]);
+
+  pool.stakedOutputTokenAmount = pool.stakedOutputTokenAmount!.minus(amount);
+  pool.inputTokenBalances[0] = pool.stakedOutputTokenAmount!;
```
Comment (steegecs) [inline subgraphs/uniswap-forks/protocols/ubeswap/src/common/handlers/handleReward.ts:56]: Same applies here

File: subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts (hunk 2 above)
Comment (sevenshi) [reply subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts:49]: i see ,the trade fee is 0.3  and LP got 0.25%  the protocol got 0.05%

File: subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts (hunk 1 above)
Comment (sevenshi) [reply subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts:16]: ok

File: subgraphs/uniswap-forks/protocols/ubeswap/src/common/handlers/handleReward.ts (hunk 4 above)
Comment (sevenshi) [reply subgraphs/uniswap-forks/protocols/ubeswap/src/common/handlers/handleReward.ts:56]: thanks

**Pull request merged**